clusters/build-clusters/common/_machinehealthcheck.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  annotations:
+  name: infra-healthcheck
+  namespace: openshift-machine-api
+spec:
+  maxUnhealthy: 2
+  nodeStartupTimeout: 10m
+  selector:
+    matchExpressions:
+    - key: machine.openshift.io/cluster-api-machine-role
+      operator: In
+      values:
+      - infra
+    - key: machine.openshift.io/cluster-api-machineset
+      operator: Exists
+  unhealthyConditions:
+  - status: "False"
+    timeout: 480s
+    type: Ready
+  - status: Unknown
+    timeout: 600s
+    type: Ready
+---
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  name: worker-healthcheck
+  namespace: openshift-machine-api
+spec:
+  maxUnhealthy: 3
+  nodeStartupTimeout: 25m
```

Comment:
Why the difference beween these settings and the ones for `infra-healthcheck`, which currently has 2 and 10m?

Reply:
Maybe we're lifting from [here][1]?  I can poke around in history...

[1]: https://github.com/openshift/managed-cluster-config/tree/0dc4c2c8d5c6215f93000da1a0990237fc77508f/deploy/osd-machine-api

Reply:
We can change the time to be a better value for sure. The current values are taken from other OSD clsuters.

Reply:
Poking around in managed history, openshift/managed-cluster-config@5912796b421e takes `nodeStartupTimeout` from 25m to 30m and [here][1] it's talking about slow `m5zn.metal` compute.  Do our build clusters run instance types that are that slow?  If not, can we leave this unset and get the default
([currently 10m][2])?

`maxUnheallthy` [currently defaults to 100%][3], but having that set to 2 (=100%?) for `infra` and 3 for `worker` seems fine.  Certainly less magical for me than a custom `nodeStartupTimeout` number.

[1]: https://github.com/openshift/managed-cluster-config/pull/1329#issue-1444558122
[2]: https://github.com/openshift/api/blob/04ea3891d9cbf7af272775d22e34935891bf36bb/machine/v1beta1/types_machinehealthcheck.go#L87
[3]: https://github.com/openshift/api/blob/04ea3891d9cbf7af272775d22e34935891bf36bb/machine/v1beta1/types_machinehealthcheck.go#L73

Reply:
I removed the `nodeStartupTimeout`